patientsearch/api.py

```diff
@@ -185,11 +186,17 @@ def external_search(resource_type, methods=["GET"]):
         current_app.logger.warn('multiple patients returned from PDMP')
     external_search_bundle['entry'][0].setdefault('id', local_fhir_patient['id'])
 
+    # TODO: add user_id to extra (available in token?)
```

Comment:
I think we _could_ get that info from the OIDC token (email from `id_token` claim) from Keycloak